README.md

```diff
@@ -43,14 +43,41 @@ The features that are currently working are the following:
 
 Requirements
 ------------------------------
-To *run* Melia, you need
-* .NET 4.5 (Mono 4 or above)
-* MySQL 5 compatible database
-
-To *compile* Melia, you need
-* C# 7 compiler, such as:
-  * [Visual Studio](http://www.visualstudio.com/en-us/products/visual-studio-express-vs.aspx) (2017 or later)
-  * [Monodevelop](http://monodevelop.com/) (With Mono version 5 or greater)
+
+### General Pre-requisites
+
+- InnoDB compatible Server such as
+  - [MySQL 5](https://dev.mysql.com/downloads/mysql/5.7.html) or [MariaDB 10](https://mariadb.org/download/)
+- Sufficient Hardware resources for the Emulator.
+  - Recommended: 1GB
+  - Minimum: 256MB
+- Sufficient Storage for the Database.
+  - Recommended: 4GB
+  - Minimum: 100MB
```

Comment:
Do you think we should make up more or less random recommendations for what the server needs, when we really have absolutely no idea? I mean, we aren't even at a point where you could accurately benchmark that yet.

Reply:
You got me on this, yes I have no idea what are the actual needs 🙈 

But I have a feeling regarding (based on previous experience) that at least for now this should suffice. (I'm actually basing these numbers on what Bukkit with a fairly PvP mod-set with MySQL would for example require).

Reply:
Though Java is memory-hungrier. And minimum 100 MB for the database? Granted, it's localhost, but my test database does not even have 2 MB. Pretty arbitrary numbers^^ But if you think they will be useful for users in some way, I don't mind having them.

Reply:
Yeah, the numbers are "kinda" arbitrary, though for real-world usages, when possibly the Emulator is closer to a real feature-set status it might... Come close or surpass those numbers. Yet, I can remove them, I don't mind 😄

Reply:
I'll merge it for now, we can always overthink it later if we want^^